tooling/noir_js_types/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "@noir-lang/types",
```

```suggestion
  "name": "@noir-lang/backend-types",
```

I don't think this is/should be a catch-all types package so maybe this is a better name?